apps/ledger-live-mobile/src/hooks/useSystem/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { useWindowDimensions, Platform } from "react-native";
+import { useSelector } from "react-redux";
+import { languageSelector, localeSelector, themeSelector } from "~/reducers/settings";
+
+const useSystem = () => {
+  // Device dimensions
+  const { width, height } = useWindowDimensions();
+
+  // Device informations
+  const os = Platform.OS;
+  const version = Platform.Version;
+
+  // Language informations
+  const language = useSelector(languageSelector);
+  const locale = useSelector(localeSelector);
+
+  // Styles
+  const theme = useSelector(themeSelector);
```

Comment:
You never use the `theme` object, am I wrong? I think that we are good with `useTheme`, no?

Reply:
This selector simply returns the current theme (black/white).